docs/sources/reference/stdlib/encoding.md

```diff
@@ -27,6 +27,29 @@ into the original string.
 tangerine
 ```
 
+## encoding.to_base64
+
+The `encoding.to_base64` function encodes a string to RFC4648-compliant base64 string
+into the original string.
```

Comment:
```suggestion
The `encoding.to_base64` function encodes a string to an RFC4648-compliant Base64 string into the original string.
```

The wording here seems awkward.  I'm not sure how to phrase it so it's still technically correct. @mattdurham could we say something along the lines of ``The `encoding.to_base64` function encodes a string to an RFC4648-compliant Base64 string and writes it into the original string.``?  Or does it overwrite the original string? Or something else?

Reply:
I hope the verbal change makes sense.